src/lsstseries/analysis/structure_function_calculators/basic_structure_function_calculator.py

```diff
@@ -0,0 +1,126 @@
+from typing import List
+
+import numpy as np
+from scipy.stats import binned_statistic
+
+from lsstseries.analysis.structure_function_argument_containers.structure_function_argument_container import (
+    StructureFunctionArgumentContainer,
+)
+from lsstseries.analysis.structure_function_calculators.structure_function_calculator import (
+    StructureFunctionCalculator,
+)
+
+
+class BasicStructureFunctionCalculator(StructureFunctionCalculator):
+    """The most rudimentary Structure Function calculation method"""
```

Comment:
I could use some help from @nevencaplar here (and for traditional documentation) in describing succinctly, what the "basic" structure function is and how it is distinguished from other implementations.

Reply:
I would avoid the use of the word `rudimentary`. I would include a reference to Equation 12 from https://arxiv.org/abs/1604.05858. I would say something like ``SF calculation method that calculates excess variance directly as a variance of observations with observational errors subtracted.

Reply:
Thanks - updated!